### DISCUSSION TITLE
Energy-Powered Rockets?

### DISCUSSION THREAD

**Discussion by Adi-0** [TeamGalacticraft/Galacticraft#231] [Feature Requests]
Hihi! First of all, I absolutely respect the amount of work ya'll have put in. I was all on board when I heard about Galacticraft: Rewoven and initially thought that the project had died down but I'm glad that this was not the case and that it just turned into Galacticraft 5!

I'm pretty sure that porting is time consuming as hell and probably just as tedious, so this is purely for conversation sake. I'm sure you guys have at least considered this once before (or maybe not who knows), but out of curiosity how do you guys view this idea? Does it fit with the goal of Galacticraft 5, or do you guys see such a rocket as too easy to use even if its walled off as an endgame item?

Regardless, thanks a ton for your work peeps and I really can't wait for the fabric release when you guys pull it off!

edit: Word choices p_p

**Comment by JustPyrrha** [comment]
I like the idea, might be something we could take a look at after the alpha.

**Comment by CLtheman1** [comment]
I think energy-powered rockets are a neat idea, and could use a technology called "electrogravitic propulsion"(look it up, it's cool) to have space launches that run without fuel. If I remember correctly, the Astro Miner with the Orion Drive already does this, so it's feasible with what's already in the mod.
